spec/controllers/api/v1/subjects_controller_spec.rb

```diff
@@ -209,6 +213,32 @@
             expect(retired).to all( be false )
           end
 
+          describe "favorited subjects" do
+            context "user has no favorites collection" do
+              it 'should return favorite as false' do
+                user.collections.destroy_all
+                get :index, request_params
+                favorites = json_response["subjects"].map{ |s| s['favorite']}
+                expect(favorites).to all( be nil )
+              end
+            end
+
+            context "user has a favorites collection" do
+              before(:each) { get :index, request_params }
```

Comment:
e.g. add collections here or move the let block down here entirely if this is where they are used.